src/worker.ts

```diff
@@ -34,8 +34,8 @@ function getImportESM () {
 
 // Look up the handler function that we call when a task is posted.
 // This is either going to be "the" export from a file, or the default export.
-async function getHandler (filename : string) : Promise<Function | null> {
-  let handler = handlerCache.get(filename);
+async function getHandler (filename : string, name : string) : Promise<Function | null> {
+  let handler = handlerCache.get(`${filename}:${name}`);
```

Comment:
```suggestion
  let handler = handlerCache.get(`${filename}/${name}`);
```

Because `:` is both a valid character in JS keys and POSIX filenames, so there’s at least some theoretical potential for collisions here?